builder.go

```diff
@@ -588,9 +588,11 @@ func (b ValExprBuilder) In(list ...interface{}) BoolExprBuilder {
 }
 
 // InTuple creates an IN expression from a tuple.
-func (b ValExprBuilder) InTuple(tuple ValTuple) BoolExprBuilder {
-	if len(tuple.Exprs) == 0 {
-		return b.makeComparisonExpr(astIn, makeErrVal("empty list"))
+func (b ValExprBuilder) InTuple(tuple Tuple) BoolExprBuilder {
+	if valTuple, ok := tuple.(ValTuple); ok {
```

Comment:
What happens if the `Tuple` is not a `ValTuple`?

Reply:
The only Tuples are ValTuple and Subquery. In ValTuple the values are specified by the caller, so you know the length before you run the query. For Subquery there isn't a way to know how many values will be returned by it, so its going to have to be ok for it to come back empty.

Reply:
Sounds good